docs/public/_redirects

```diff
@@ -7,3 +7,4 @@ https://vitejs.dev/* https://vite.dev/:splat 301!
 /guide/api-vite-environment.html /guide/api-environment 302
 /guide/comparisons /guide/why#how-vite-relates-to-other-unbundled-build-tools 302
 /guide/comparisons.html /guide/why#how-vite-relates-to-other-unbundled-build-tools 302
+/rolldown /guide/rolldown 301
```

Comment:
I wonder if we should have a prefix so that we don't conflict with normal pages (e.g. `/s/rolldown`, `/_rolldown`).

Reply:
That would defeat the purpose. I think if there is later a rolldown top level page, it would be ok to end up there. I would prefer not to do this often though, as anchors may be broken later

Reply:
That's true. 👍